wger/settings_global.py

```diff
@@ -310,7 +329,7 @@
 #
 # Django compressor
 #
-STATIC_ROOT = ''
+STATIC_ROOT = BASE_DIR
```

Comment:
I don't think we should be overriding the value for STATIC_ROOT set in line 189.